src/components/templates/Jazzbar/JazzTab/JazzTab.tsx

```diff
@@ -164,10 +164,7 @@ const Jazz: React.FC<JazzProps> = ({ setUserList, venue }) => {
       )}
 
       {!seatedAtTable && parentVenue && (
-        <BackButton
-          onClick={backToParentVenue}
-          locationName={parentVenue.name}
-        />
+        <BackButton onClick={backToParentVenue} />
```

Comment:
Why was `locationName` removed here? We want to keep this on the main platform, it should only be removed on `env/github`.

Reply:
fixed in 088c807569a37b13ce953e79d623431827421b6a